app.js

```diff
@@ -125,37 +125,22 @@ app.get('/data/bootstrapcdn.json', function (req, res) {
 
 // Merge in variable options for sitemap.
 function sitemapOptions (options) {
-    options.http = 'https';
-
     if (env === 'production') {
```

Comment:
Yeah but lower. This specific check does nothing alone. How about merging
with the code that follows and negate the check?

On Aug 12, 2016 23:24, "Joshua Mervine" <notifications@github.com> wrote:

> In app.js
> <https://github.com/MaxCDN/bootstrap-cdn/pull/737#discussion_r74655048>:
>
> > @@ -125,37 +125,22 @@ app.get('/data/bootstrapcdn.json', function (req, res) {
> >
> >  // Merge in variable options for sitemap.
> >  function sitemapOptions (options) {
> > -    options.http = 'https';
> > -
> >      if (env === 'production') {
>
> The check adds the disallow to the robots.txt and that's it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MaxCDN/bootstrap-cdn/pull/737/files/7c380cfac434705b88356aa66477d0ab531e41fb..9ef2f34d397eeb9e96cfb282d1306e9fb431583a#r74655048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtXtgyg7kq30q-lUpfz_QELhF5a7wks5qfNaUgaJpZM4JeeRW>
> .
>